<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshSettingsPage</class>
 <widget class="QWidget" name="SshSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KMessageWidget" name="errorMessageWidget">
     <property name="messageType">
      <enum>KMessageWidget::Error</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KMessageWidget" name="successMessageWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="deviceLabel">
        <property name="text">
         <string>Device:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ipLabel">
        <property name="text">
         <string>Hostname / IP:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="ipLineEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="usernameLineEdit"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="sshOptionsLabel">
        <property name="text">
         <string>SSH Options:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="sshOptionsLineEdit"/>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="copyKeyButton">
        <property name="text">
         <string>Copy SSH Key to device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="multiConfigWidget" native="true"/>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="testButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Test connection and save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>SSH Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>SSH_ASKPASS:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="sshaskpassLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
